libraries/adaptive-expressions/src/builtinFunctions/xml.ts

```diff
@@ -28,48 +28,24 @@ export class XML extends ExpressionEvaluator {
     }
 
     private static platformSpecificXML(args: unknown[]): { value: unknown; error: string } {
-        if (typeof window !== 'undefined' || typeof self !== 'undefined') {
-            // this is for evaluating in browser environment, however it is not covered by any test currently
-            // x2js package can run on browser environment, see ref: https://www.npmjs.com/package/x2js
-            // eslint-disable-next-line @typescript-eslint/no-var-requires
-            const X2JS = require('x2js');
-            let result: unknown;
-            let error: string;
-            let obj: unknown;
-            try {
-                if (typeof args[0] === 'string') {
-                    obj = JSON.parse(args[0] as string);
-                } else if (typeof args[0] === 'object') {
-                    obj = args[0];
-                }
-
-                result = new X2JS.json2xml_str(obj);
-            } catch (err) {
-                error = `${args[0]} is not a valid json`;
+        let result: unknown;
+        let error: string;
+        let obj: unknown;
+        try {
+            if (typeof args[0] === 'string') {
+                obj = JSON.parse(args[0] as string);
+            } else if (typeof args[0] === 'object') {
+                obj = args[0];
             }
-
-            return { value: result, error: error };
-        } else {
-            // xml2js only support node environment, see ref: https://github.com/Leonidas-from-XIV/node-xml2js
-            let result: unknown;
-            let error: string;
-            // eslint-disable-next-line @typescript-eslint/no-var-requires
-            const xml2js = require('xml2js');
-            let obj: unknown;
-            try {
-                if (typeof args[0] === 'string') {
-                    obj = JSON.parse(args[0] as string);
-                } else if (typeof args[0] === 'object') {
-                    obj = args[0];
-                }
-
-                const builder = new xml2js.Builder();
-                result = builder.buildObject(obj);
-            } catch (err) {
-                error = `${args[0]} is not a valid json`;
-            }
-
-            return { value: result, error: error };
+            const parser = new j2xParser({
+                indentBy: '  ',
+                format: true,
+            });
+            result = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>\n' + parser.parse(obj).trim('\n');
```

Comment:
We try to use string interpolation when possible (works with multiline strings). Try the following:

```typescript
result = `<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
  ${parser.parse(obj)}`.trim('\n');
```

Reply:
Sure I will change it. Also argument \n for trim is not needed. As trim automatically removes new lines at the end.